src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
No, I wasn't planning on investigating anything. I was just trying to review a change. Maybe @ryanzim is interested in helping?